### PR TITLE
add pageNumber to path, fixed disabled buttons

### PR DESCRIPTION
**Pull request by BogdanskiJ**
I just added working pageNuber in url path and fixed disabled buttons.

### CODE REVIEW

**Review by jakubkedzierski92 — APPROVED** [review]
well done! A huge progress

File: src/features/movieList/Movie/index.js [hunk 1]
```diff
@@ -9,11 +9,10 @@ const Movie = ({ movieTitle, movieRating, votesNumber, movieYear, moviePosterApi
     if (a === b.id) { return (b.name) }
   }
   const url_img = "https://image.tmdb.org/t/p/w342/"
-  console.log(id)
+
```
Comment (Werka481) [inline src/features/movieList/Movie/index.js:12]: I think you can delete console.log.

**Review by Werka481 — APPROVED** [review]
Pagination looks great. Everything works. Great job!

File: src/features/movieList/Movie/index.js (hunk 1 above)
Comment (Kubabogd) [reply src/features/movieList/Movie/index.js:12]: Ok. I delete this. Thanks :)

**Pull request merged**